about/terms_of_service.rst

```diff
@@ -1,326 +1,504 @@
+For Enterprise customers only, Mailsac shall maintain, and shall implement when
+necessary, a written business continuity plan (“BCP”) that permits it to
+recover from a triggering event and continue providing Services to user within
+the specified recovery time objectives. The BCP will be written and maintained
+in the Mailsac Information Assurance Program, available upon request to
+Enterprise customers.
 
 18. Indemnification
+-------------------
 
-Each party agrees to indemnify, defend, and hold harmless the other party from and against any loss, cost, or damage of any kind (including reasonable outside attorneys’ fees) to the extent arising out of its breach of this Agreement, and/or its negligence or willful misconduct.
+Each party agrees to indemnify, defend, and hold harmless the other party from
+and against any loss, cost, or damage of any kind (including reasonable outside
+attorneys’ fees) to the extent arising out of its breach of this Agreement,
+and/or its negligence or willful misconduct.
 
 19. Insurance
+-------------
 
-The parties will, at their own cost and expense, obtain and maintain in full force and effect during the term of this Agreement the following insurance coverages:
+The parties will, at their own cost and expense, obtain and maintain in full
+force and effect during the term of this Agreement the following insurance
+coverages:
 
 - Commercial General Liability
 - Cyber Security Liability
 
 20. Other Agreements
 --------------------
-If a clause of this agreement is found to be invalid or violated, the rest of this agreement still stands.
 
-This agreement represents the entire agreement between the user and Forking Software LLC, which includes the Privacy Policy. The two parties may supersede parts of this agreement through writing signed by legal representatives of both parties. Clauses of this agreement not addressed in any superseding agreement will still stand.
+If a clause of this agreement is found to be invalid or violated, the rest of
+this agreement still stands.
+
+This agreement represents the entire agreement between the user and Forking
+Software LLC, which includes the Privacy Policy. The two parties may supersede
+parts of this agreement through writing signed by legal representatives of both
+parties. Clauses of this agreement not addressed in any superseding agreement
+will still stand.
+
+Both parties may assign this agreement, or any rights or obligations hereunder,
+including the right to enforce all of its terms,
 
-Both parties may assign this agreement, or any rights or obligations hereunder, including the right to enforce all of its terms, (i) in connection with a merger, acquisition, corporate reorganization, change in control or sale of all or substantially all of its assets or (ii) to an Affiliate (that is an Affiliate at the time of such assignment).
+1. in connection with a merger, acquisition, corporate reorganization, change
```

Comment:
Formatting seems broken here.

Reply:
Can you explain how the formatting is broken?